cookie-banner-rules-list.json

```diff
@@ -636,6 +636,265 @@
       },
       "id": "c5243e7c-eb86-4a9d-947c-7129e99fbd72",
       "last_modified": 1660836869997
-    }
+    },
+    	{
+  "click": {
+    "optIn": "button#onetrust-accept-btn-handler",
+    "optOut": "button#onetrust-reject-all-handler",
+    "presence": "div#onetrust-consent-sdk"
+  },
+  "domain": "cloudflare.com",
+  "schema": 1665622072973,
+  "cookies": {},
+  "id": "0b42c238-b54d-4106-8d9a-81df5bc0b3ae"
+  },
+    {
```

Comment:
nit: indentation, can we align this with the bracket above?

Reply:
Fixed indentation for all the new rules.